lib/frozen_record/scope.rb

```diff
@@ -229,7 +238,13 @@ def offset!(amount)
       self
     end
 
+    def to_hashable_string
+      "#{@klass} -- WHERE: #{@where_values.uniq.sort}, NOT: #{@where_not_values.uniq.sort}, ORDER_BY: #{@order_values.uniq},
```

Comment:
I'm not a fan of this implementation. Relying on the objects `inspect` is brittle and could easily go wrong.

I'd rather make `where_values`, `where_not_values`, `order_values` etc protected, and compare/hash them directly.